cider/function/scalar/RuntimeFunctions.cpp

```diff
@@ -198,6 +198,25 @@ DEF_SAFE_INF_DIV_NULLABLE(double, double, safe_inf_div)
 #undef DEF_ARITH_NULLABLE
 #undef DEF_SAFE_INF_DIV_NULLABLE
 
+extern "C" ALWAYS_INLINE void* extract_arrow_array_null(int8_t* arrow_pointer,
+                                                        int64_t index) {
+  ArrowArray* array = reinterpret_cast<ArrowArray*>(arrow_pointer);
+  auto null_data = array->children[index]->buffers[0];
+  return const_cast<void*>(null_data);
+}
+
+extern "C" ALWAYS_INLINE void* extract_arrow_array_data(int8_t* arrow_pointer,
+                                                        int64_t index) {
+  ArrowArray* array = reinterpret_cast<ArrowArray*>(arrow_pointer);
```

Comment:
seems same as above? use` extract_arrow_array_buffer_at(int8_t* arrow_pointer, int64_t index)`?

Reply:
OK, that's also make sense